pkgs/top-level/all-packages.nix

```diff
@@ -240,6 +240,8 @@ with pkgs;
 
   alire = callPackage ../development/tools/build-managers/alire { };
 
+  alp = callPackage ../by-name/al/alp/package.nix { };
```

Comment:
??

Reply:
yikes that accept commit button seems to have worked in this case, corrected.